ui/components/app/tab-bar/index.scss

```diff
@@ -53,6 +56,17 @@
           min-height: 14px;
         }
       }
+
+
+      &__icon {
+        margin-right: 16px;
+      }
+
+      &__title {
+        @include Paragraph;
+
+        font-size: 18px;
```

Comment:
We should use `1.125rem` instead of pixel font sizing.

Reply:
Ok.